github/libp2p/repository_collaborator.json

```diff
@@ -472,1272 +128,377 @@
     "Stebalien": {
       "permission": "admin"
     },
-    "aarshkshah1992": {
-      "permission": "push"
-    },
-    "hsanjuan": {
-      "permission": "admin"
-    },
```

Comment:
I would also like to maintain my permissions as they are in general, particularly those in the repositories where I'm the main and only maintainer.